meson.build

```diff
@@ -122,22 +122,29 @@ wx_dep = dependency('wxWidgets', version: wx_minver,
 if wx_dep.found()
     deps += wx_dep
 else
-    build_shared = 'ON'
+    build_shared = true
     if get_option('default_library') == 'static'
-        build_shared = 'OFF'
+        build_shared = false
     endif
     build_type = 'Release'
     if get_option('buildtype') == 'debug'
         build_type = 'Debug'
     endif
 
-    wx = cmake.subproject('wxWidgets', cmake_options: ['-DwxBUILD_INSTALL=OFF',
-                                                       '-DwxBUILD_PRECOMP=OFF', # otherwise breaks project generation w/ meson
-                                                       '-DwxBUILD_SHARED=@0@'.format(build_shared),
-                                                       '-DwxUSE_WEBVIEW=OFF', # breaks build on linux
-                                                       '-DCMAKE_BUILD_TYPE=@0@'.format(build_type),
-                                                       '-DwxUSE_IMAGE=ON',
-                                                       '-DwxBUILD_MONOLITHIC=ON']) # otherwise breaks project generation w/ meson
+    opt_var = cmake.subproject_options()
+    opt_var.add_cmake_defines({
+        'wxBUILD_INSTALL': false,       # '-DwxBUILD_INSTALL=OFF'
+        'wxBUILD_PRECOMP': false,       # '-DwxBUILD_PRECOMP=OFF', # otherwise breaks project generation w/ meson
+        'wxBUILD_SHARED': build_shared, # '-DwxBUILD_SHARED=@0@'.format(build_shared)
+
+        'wxUSE_WEBVIEW': false,         # '-DwxUSE_WEBVIEW=OFF', # breaks build on linux
+        'CMAKE_BUILD_TYPE': build_type, # '-DCMAKE_BUILD_TYPE=@0@'.format(build_type)
+        'wxUSE_IMAGE': true,            # '-DwxUSE_IMAGE=ON'
+        'wxBUILD_MONOLITHIC': true      # '-DwxBUILD_MONOLITHIC=ON', # otherwise breaks project generation w/ meson
```

Comment:
You can get rid of the comments with the old options, just leave the notes like "breaks build on linux". The new syntax is more readable anyway, I just hadn't gotten around to converting it over.